homeassistant/components/energy/sensor.py

```diff
@@ -214,7 +216,7 @@ def __init__(
             f"{config[adapter.entity_energy_key]}_{adapter.entity_id_suffix}"
         )
         self._attr_device_class = DEVICE_CLASS_MONETARY
-        self._attr_state_class = STATE_CLASS_MEASUREMENT
+        self._attr_state_class = STATE_CLASS_TOTAL
```

Comment:
With TOTAL the frontend needs to know if it should take the increasing or decreasing part of the stat. Do we need a migration?

Reply:
Or is SUM fine, because we want the total "cost", which is net anyway?

Reply:
Yeah, for cost sensors we just need the total sum.